bitcoin/src/crypto/ecdsa.rs

```diff
@@ -174,7 +174,8 @@ impl fmt::LowerHex for SerializedSignature {
         fmt::LowerHex::fmt(&(**self).as_hex(), f)
     }
 }
-impl_to_hex_from_lower_hex!(SerializedSignature, |signature: &SerializedSignature| signature.len * 2);
+impl_to_hex_from_lower_hex!(SerializedSignature, |signature: &SerializedSignature| signature.len
+    * 2);
```

Comment:
In 9a5ba9b6df1c37e6f6e29029f3169a9ba8d1ceaa:

ugh

Reply:
yuck